apps/backend/src/datasources/stfc/StfcUserDataSource.ts

```diff
@@ -347,7 +347,18 @@ export class StfcUserDataSource implements UserDataSource {
       (stfcBasicPerson) => (stfcBasicPerson ? toEssUser(stfcBasicPerson) : null)
     );
   }
+  async getUsersByUserNumbers(ids: number[]) {
+    const stfcBasicPersonDetails = await this.getStfcBasicPeopleByUserNumbers(
+      ids.map((id) => id.toString())
+    );
+    const stfcUsers = stfcBasicPersonDetails
+      ? stfcBasicPersonDetails.map((stfcbasicPerson) =>
+          toEssUser(stfcbasicPerson)
+        )
+      : null;
```

Comment:
We might want to remove the ternary operator which returns null in false case. The definition for stfcBasicPersonDetails type is stfcBasicPersonDetails[] here which can only return array type and will never return falsy value because empty array is always true.